com.visualstudio.code.oss.json

```diff
@@ -16,7 +16,8 @@
         "--device=dri",
         "--filesystem=host",
         "--persist=.vscode-oss",
-        "--talk-name=org.freedesktop.Notifications"
+        "--talk-name=org.freedesktop.Notifications",
+        "--env=GTK_IM_MODULE=xim"
```

Comment:
@TingPing I'm not an input method expert, will this break the built in ibus support?

Reply:
I don't know whether it breaks, but I just discovered that the IME won't work with our without this environment variable :joy: :joy: @TingPing What should we do to solve this problem? 🤔 https://github.com/flathub/com.visualstudio.code/issues/24 and https://github.com/flathub/io.atom.Atom/issues/6

Reply:
ibus seems to be the only supported IME within flatpak currently. But just to clarify, are you saying that ibus doesn't work either? As mentioned in https://github.com/flathub/org.signal.Signal/issues/17 could you try building and using [the ibus testcase flatpak](https://github.com/flathub/org.signal.Signal/issues/17#issuecomment-368955752)?

Reply:
IME works in the testcase flatpak. What should I add to my recipe? :thinking:

Reply:
Not sure currently - but this very much seems to be a common issue for all electron apps.

Reply:
:woman_facepalming: